Core.Auditing/IfxAuditLogger.cs

```diff
@@ -59,16 +59,21 @@ public void LogApplicationAuditEvent(ITelemetryClient telemetryClient, AuditMand
             }
         }
 
-        public void LogTokenGenerationAuditEvent(ITelemetryClient telemetryClient, OperationResult operationResult, TargetResource[] targetResources, CallerIdentity[] callerIdentities, AuditOptionalProperties auditOptionalProperties = null)
+        public void LogTokenGenerationAuditEvent(ITelemetryClient telemetryClient, OperationResult operationResult, TargetResource[] targetResources, CallerIdentity[] callerIdentities, string operationName, AuditOptionalProperties auditOptionalProperties = null)
         {
-            LogAuthorizationAuditEvent(telemetryClient, operationResult, TokenGenerationOperation, targetResources, callerIdentities, auditOptionalProperties);
+            LogAuthorizationAuditEvent(telemetryClient, operationResult, operationName, targetResources, callerIdentities, auditOptionalProperties);
```

Comment:
>operationName [](http://example.com/codeflow?start=73&length=13)

if we change above to tokenType, then here, we can concatenate TokenGenerationOperation constant with tokenType.